Sum-and-Number-of-Integers.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
```

Comment:
your output - Enter a string of integers separated by spaces: 54 rt 67
                      Sum of integers: 121
                      Total number of integers: 3
expected output - Enter a string of integers separated by spaces: 54 rt 67
                              Sum of integers: 121
                              Total number of integers: 2